packages/@okta/courage-for-signin-widget/package.json

```diff
@@ -7,9 +7,9 @@
     "build": "webpack"
   },
   "devDependencies": {
-    "@okta/babel-plugin-handlebars-inline-precompile": "3.1.0-alpha.3660.g8faa6ac",
-    "@okta/eslint-plugin-okta-ui": "0.1.0-alpha.3660.g8faa6ac",
-    "@okta/courage": "4.6.0-alpha.3660.g8faa6ac",
+    "@okta/babel-plugin-handlebars-inline-precompile": "3.1.0-beta.4200.gf3d0a27",
+    "@okta/courage": "4.6.0-beta.4200.gf3d0a27",
+    "@okta/eslint-plugin-okta-ui": "0.1.0-beta.4181.g1f38f27",
```

Comment:
Needed. Otherwise can't do `yarn build` from within `courage-for-signin-widget` (since the alpha versions are not available anymore). cc: @aarongranick-okta